fms_fsdp/config/training.py

```diff
@@ -59,3 +59,13 @@ class train_config:
 
     # compile
     use_torch_compile: bool = True
+
```

Comment:
Are we re-using all/most of the above for Speculator training or would it make more sense to have a separate speculator training configuration?

Reply:
Separate config not needed. Many of the above are bring used, and the remaining ones are also compatible.